### PR TITLE
fix flow issue and incorrect object.keys wrapping

### PR DESCRIPTION
**Pull request by chrisnojima**
fixes https://github.com/keybase/client/issues/14432
@keybase/react-hackers

### CODE REVIEW

**Comment by mmaxim** [comment]
Would this have caused other problem with loading avatars? We have two tickets on core about how they failing to load in some circumstances.

**Comment by chrisnojima** [comment]
Mmm unlikely

File: shared/common-adapters/avatar.js [hunk 1]
```diff
@@ -109,6 +109,8 @@ class SharedAskForUserData {
   _userQueue = {}
   _userLastReq = {}
   _username = ''
+  _teamQueue: Object = {}
+  _userQueue: Object = {}
```
Comment (mmaxim) [inline shared/common-adapters/avatar.js]: Both of these already in there, this one just has a type on it. Delete the two above it?

File: shared/common-adapters/avatar.js [hunk 2]
```diff
@@ -143,12 +143,10 @@ class SharedAskForUserData {
       }
       return false
     })
-    this._teamQueue = {}
```
Comment (mmaxim) [inline shared/common-adapters/avatar.js]: I don't get why this is removed. How does this ever get cleared?

**Comment by mmaxim** [comment]
One general concern about this entire avatar strategy here is what happens if the service fails to load an avatar. It seems like that failure is going to be stuck on the frontend for 30 minutes. There isn't any kind of feedback into this `SharedAskForUserData` of whether or not the requests succeeded, it just marks the time for when it made the request. 

Maybe we could reconsider this whole thing, and go back to just hitting the service every time (which does have a story for a failed avatar load). I'd be curious to know just what kind of speed this thing brings, given the complexity it brings to the table.

**Comment by mmaxim** [comment]
The cache also only really seems to affect the case where I am just scrolling the inbox up and down (probably an unlikely user interaction). The first load is still hitting the service for everything (given its been 30 minutes), so I am still sending a call per avatar if I am interacting with the app in the common case. Since that is true, why not just request every time and eliminate a whole class of bugs from ever happening?

**Comment by chrisnojima** [comment]
i'd prefer to not discuss architecture discussions in a bugfix pr

**Comment by mmaxim** [comment]
K I'm going to put up a different PR with some change I think will help here.

**Pull request merged**